lab-cortney/lib/greet.js

```diff
@@ -0,0 +1,6 @@
+'use strict';
+
+module.exports = function(name) {
+	const input = process.argv[2] || name;
```

Comment:
I usually try to keep all access to process.argv in the main program, it helps make the program more readable and also helps keep things  organized.

i usualy parse process.arv and store the data i need in a variable then pass those variables into my my modules :)